py/foot.py

```diff
@@ -1,19 +1,30 @@
 import dominate
 from dominate.tags import *
 
-def _foot():
-    """Returns the generated HTML footer. TODO: edit this description
+logo_img = 'https://drive.google.com/uc?export=view&id=1gq2ARk8HOCKw9MBrUfrEYe3GJjHGyBta'
```

Comment:
since we're having all of these links for images, can we create a constants file and make the corresponding variables in all caps? see: https://betterprogramming.pub/does-python-have-constants-3b8249dc8b7b

Reply:
Aight done!